db/schema.rb

```diff
@@ -212,7 +212,7 @@
     t.integer  "client_data_id"
     t.string   "client_data_type", limit: 255
     t.integer  "requester_id"
-    t.string   "public_id",        limit: 255
```

Comment:
imo, the prod db is canonical. My guess is that we have different versions of psql in cloud.gov vs local dev and that is throwing off the syntax slightly.

My suggestion: compare your local psql schema against this:
```
capc2-prod=# \d proposals
                                        Table "public.proposals"
      Column      |            Type             |                       Modifiers                        
------------------+-----------------------------+--------------------------------------------------------
 id               | integer                     | not null default nextval('proposals_id_seq'::regclass)
 status           | character varying(255)      | 
 created_at       | timestamp without time zone | 
 updated_at       | timestamp without time zone | 
 client_data_id   | integer                     | 
 client_data_type | character varying(255)      | 
 requester_id     | integer                     | 
 public_id        | character varying(255)      | 
 visit_id         | uuid                        | 
Indexes:
    "proposals_pkey" PRIMARY KEY, btree (id)
    "index_proposals_on_client_data_id_and_client_data_type" btree (client_data_id, client_data_type)
Foreign-key constraints:
    "fk_rails_c557d6413c" FOREIGN KEY (visit_id) REFERENCES visits(id)
    "requester_id_fkey" FOREIGN KEY (requester_id) REFERENCES users(id)
Referenced by:
    TABLE "attachments" CONSTRAINT "proposal_id_fkey" FOREIGN KEY (proposal_id) REFERENCES proposals(id)
    TABLE "comments" CONSTRAINT "proposal_id_fkey" FOREIGN KEY (proposal_id) REFERENCES proposals(id)
    TABLE "proposal_roles" CONSTRAINT "proposal_id_fkey" FOREIGN KEY (proposal_id) REFERENCES proposals(id)
    TABLE "steps" CONSTRAINT "proposal_id_fkey" FOREIGN KEY (proposal_id) REFERENCES proposals(id) ON DELETE CASCADE
```

if they are identical, then there is something else going on. Otherwise, run the appropriate `alter` sql commands to get your local db in sync with the schema above, and then run `rake db:schema:dump` and `git diff db/schema.rb` and see if anything shows up. If nothing differs, then we have some other problem.

It's not a big deal; just wanted to note somewhere that I keep seeing that noise in the PRs for schema.db and why it is happening.